src/conversion/conversion.js

```diff
@@ -102,6 +114,347 @@ export default class Conversion {
 		};
 	}
 
+	/**
+	 * Sets up converters between the model and the view which convert a model element to a view element (and vice versa).
+	 * For example, model `<paragraph>Foo</paragraph>` is `<p>Foo</p>` in the view.
+	 *
+	 * `definition.model` is a `String` with a model element name to converter from/to.
+	 *
+	 *		// Simple conversion from `paragraph` model element to `<p>` view element (and vice versa).
+	 *		conversion.elementToElement( { model: 'paragraph', view: 'p' } );
+	 *
+	 *		// Override other converters by specifying converter definition with higher priority.
+	 *		conversion.elementToElement( { model: 'paragraph', view: 'div', priority: 'high' } );
+	 *
+	 *		// View specified as an object instead of a string.
+	 *		conversion.elementToElement( {
+	 *			model: 'fancyParagraph',
+	 *			view: {
+	 *				name: 'p',
+	 *				class: 'fancy'
+	 *			}
+	 *		} );
+	 *
+	 *		// Use `upcastAlso` to define other view elements that should be also converted to `paragraph` element.
+	 *		conversion.elementToElement( {
+	 *			model: 'paragraph',
+	 *			view: 'p',
+	 *			upcastAlso: [
+	 *				'div',
+	 *				{
+	 *					// Any element with `display: block` style.
+	 *					name: /./,
```

Comment:
I think we should improve it, like we did for attributes in https://github.com/ckeditor/ckeditor5-engine/issues/1239. It should be no `name` in this case.

Reply:
This is a tricky example, actually.

First of all, for `Matcher` purposes, specifying something like:

```
{
	name: true,
	class: ...
}
```

is just stupid, cause the name could be omitted. It says "any name" and every element has a name.

OTOH, the `name` is in upcast definition intentionally, so that `name` is returned in a match so it will be consumed.

I think that if we want to improve that, we should add `name` to matches object (the one returned by `Matcher#match`) in `elementToElement` and `attributeToElement` (so whenever there is a view element is converted.

Reply:
Okay I did as described above ☝️